spec/recipes/server_spec.rb

```diff
@@ -85,6 +85,41 @@
     include_examples 'allow weak hmacs'
   end
 
+  context 'with permit_tunnel set to true' do
```

Comment:
@bobchaos can you please move them, in order to have them after [this](https://github.com/dev-sec/chef-ssh-hardening/blob/3b7f92f22d5dddce1a5baf8027bc76a8a8b2e834/spec/recipes/server_spec.rb#L227) line?

can you also please embed them in its own `describe 'permit_tunnel option'` block and work with let in order to set/change the attribute in the different contexts. You can take [this](https://github.com/dev-sec/chef-ssh-hardening/blob/3b7f92f22d5dddce1a5baf8027bc76a8a8b2e834/spec/recipes/server_spec.rb#L273-L308) as example, please have a look to the `use_pam` variable